### ISSUE TITLE
Annotation / discussion / tl;dr tool for public proposals 

### ISSUE THREAD

**Issue by dylanjbarth** [sketch-city/project-ideas#108]
## Context
I recently watched [Net Neutrality II: Last Week Tonight with John Oliver](https://www.youtube.com/watch?v=92vuuZt7wak). (tl;dw: FCC wants to change laws to potentially allow ISPs to slow Internet traffic to sites that don't pay them 💸 ). He urges people to take action by commenting on the FCC proposal by visiting [gofccyourself.com](gofccyourself.com), which redirects you to https://www.fcc.gov/ecfs/search/proceedings?q=name:((17-108)) and then there is a scary form you have to fill out to leave a public comment on the proposal to change net neutrality rules. 

I didn't really know what to say off the top of my head, so I googled around for help and found [How to comment on the FCC’s proposal to revoke net neutrality](https://techcrunch.com/2017/04/27/how-to-comment-on-the-fccs-proposal-to-revoke-net-neutrality/). 

This bit drew my attention: 
> All the comments will be looked at in one way or another, but substantial ones that address specific points in the proposal will almost certainly get a little more attention. 

The [proposal](https://apps.fcc.gov/edocs_public/attachmatch/DOC-344614A1.pdf) is 58 pages long!!! I seriously doubt that people without some level of subject matter expertise are going to read through it. But I would love to have a version where I could see annotations or even just concise summaries (tldr's) by subject matter experts. 

I think this could go a couple different directions, so breaking out a few problems / ideas below. 

## Problem Statements
1. Public proposals are long, boring, and full of details, and therefore hard for an average voter to understand and interpret. 
2. The format for collecting feedback on these proposals seems to uh... suck. It's a multi-step complicated process to leave a comment, it seems super government-y and formal, and ...
3. It feels like a black hole, ie it's not clear to me how my feedback will contribute to the discussion. I'd like to see a more public discussion board, with crowd curation of high quality discourse. 

## Ideas / applications
1. How about a mash up of google docs style commenting directly on the document, combined with the discussion tools of [discourse](https://www.discourse.org/features), and the curation system of Reddit 😁  The wikipedia of public proposals. 
2. Provide a Quora like platform for SME's to create tl;dr's of public proposals. Probably less possible but potentially fun, could we use deep learning to automatically summarize? [[1]](https://www.quora.com/Has-Deep-Learning-been-applied-to-automatic-text-summarization-successfully)
3. I feel like a tool along these lines could be useful as well for *consumer advocacy* -- Terms of Service share a lot of similar characteristics to public proposals, long, full of legalese, etc. 

## Open questions
* Does anyone know of a tool like this? I expected a top hit when I googled "summaries for public proposals" or "annotation tool for public documents" but I didn't find much. 
* Any other potential ideas / applications / related problems? 
* Is anyone interested in scoping this to a hackathon sized project?

**Comment by ghost** [comment]
It sounds like Google Docs would be a good application to test the value of this! For the hackathon, this could involve less platform-building so more effort can be spent on annotating a few proposals to showcase/doing a quick study to test its effectiveness. Seems like a good project for people with legal experience!

You can also use https://genius.com/ as an inspiration for a feature set ([example](https://genius.com/Nicki-minaj-super-bass-lyrics)) since they're doing very similar annotations and analysis with a crowd perspective, but for song lyrics.

A totally off-topic application for this would be doing the same thing for research papers, to make research more accessible to the public (but without totally butchering them with clickbait headlines)

**Comment by LawExplorer** [comment]
This [page](https://www.fcc.gov/consumers/guides/how-comment#top) of the website of the FCC contains information about the procedure for comments, including a glossary of terms and phrases.  (accessed May 11, 2017)

**Comment by Hzuni** [comment]
This project sounds really cool! Would you know of where I could look at project proposals specific for the City of Houston? I'd like to get a sense of just how bad the legalese is(I'm not a legal person at all). 

When thinking about this problem, I would picture that each project proposal would need a good base line starting point in order for even legal people to get interested in bothering to read the whole proposal. Local newspapers already do summaries to help uninformed voters and could maybe be used to generate new posts on the platform? Like say whenever a local news reports on a new proposal that's out there the platform could create a new post and people can read that summary and start adding there own interpretations from there if they think the proposal is about an important issue which is relevant to them.

**Comment by LawExplorer** [comment]
The availability government information in plain language is a big issue, and a number of resources are available in print and online.  One of those is [PlainLanguage.gov](http://www.plainlanguage.gov/).

**Comment by dylanjbarth** [comment]
@cdruecker-aetikos https://genius.com/ is probably a good starting point when thinking about the user experience, thanks! and I really like the research paper use case as well. 

@LawExplorer thanks for those resources. Seems like many government bodies have committed to communicating in plain language, eg https://www.fcc.gov/general/plain-writing-fcc and a full list here http://www.plainlanguage.gov/plLaw/fedGovt/index.cfm. My takeaway would be that even if something is communicated plainly, it can still be clearer and more meaning can be uncovered through discussion and annotation by SMEs. 

@Hzuni I agree with you about the base line. I could see journalists being power users of this tool, uploading project proposals initially and kicking off discussions with their initial annotations. 

Re Houston specific proposals, @fileunderjeff might have some ideas of where we could find some examples?

**Comment by nhahuynh** [comment]
Here is a good example of how the layout could be from a Texas Tribune article explaining the bathroom bill in plain English: https://apps.texastribune.org/texas-bathroom-bill-annotated/
What about an upvote system like on Reddit? And if we're relying on SME's , we need to create a vetting system to make sure trolls won't spam the threads.

**Comment by LawExplorer** [comment]
Another possible model is the layout for the caselaw summaries of LexisNexis.  This tutorial demonstrates the manner in which those are organized: [http://www.lexisnexis.com/tutorial/global/globaltutorial_frameset.asp?sPage=summaries&adaptation=academic&lbu=US&locale=en_us](http://www.lexisnexis.com/tutorial/global/globaltutorial_frameset.asp?sPage=summaries&adaptation=academic&lbu=US&locale=en_us).

**Comment by dylanjbarth** [comment]
I chatted with a guy named Filip on Saturday morning at the hackathon, and he pointed me to a few more relevant previous projects I wanted to share here: 

* Sunlight Labs previous projects (some interesting outbound links there -- sunlight labs have shut down but their projects all have new owners) 
* http://www.billtracker.org/ -- this was an Austin Hack For Change hackathon project that Filip participated in -- it's by far the closest thing to this idea I've seen to what I was initially imagining, but there is so much more that could be done! 

Their contact form fails to submit, but I'm going to try and get connected through Filip. Their source code is https://github.com/mcdermottsolutions/bill-tracker and they have a list of issues, but activity has been pretty stagnant for the past year. Regardless, it looks like a fantastic starting place!
